website/docs/guides/orchestration/running-dbt-cloud-jobs/3-anti-patterns.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Anti Patterns for dbt Cloud jobs"
+id: "dbt-cloud-job-anti-patterns"
+slug: 3-anti-patterns
+description: Recognize common anti-patterns in dbt Cloud jobs
+displayText: dbt Cloud job anti-patterns
+hoverSnippet: Recognize common anti-patterns in dbt Cloud jobs.
+---
+
+dbt's ref() function enables running data models in the proper sequence automatically and in parallel. Thus, it's possible to make things harder than they need to be by making dbt Cloud job runs too specific. See organizing-dbt-cloud-jobs for our recommendations on approach.
+
+### Example DAG
+
+Model `fct_orders` ref()s `int_orders` which ref()s `stg_orders`
+
+The `models/` directory of the dbt Project looks like this:
+
+```
+models/
+  staging/
+    stg_orders.sql
+    stg_items.sql
+  intermediate/
+    int_orders.sql
+    int_order_items.sql
+  marts/
+    fct_order_items.sql
+```
+
+### Creating and running a separate dbt Cloud job for each model
+
+In Job 1:
+`dbt run -s stg_orders`
+
+In Job 2:
+`dbt run -s int_orders`
+
+In Job 3:
+`dbt run -s fct_orders`
+
+Creating distinct jobs to invoke a single model as shown in the above example is strongly discouraged because it creates a lot of overhead for each job run. Specifically for each job:
+- the run will queue to be processed by the dbt Cloud Scheduler
+- the dbt Cloud Scheduler will prepare an environment to execute dbt in
+- dbt will clone the repository
+- dbt will clone the packaage dependencies
+- dbt will parse the complete project
+- dbt will establish the connection with the warehouse
+all before the model build begins.
+This setup cost is efficient when dbt runs a large swath of the DAG, but every inefficient if dbt runs a single model at a time. dbt is optimized to process a graph of nodes, and it's important to not extract each model from the DAG separately in a job.
```

Comment:
This is all true, but it's more a symptom of the bigger reason. All of that is inefficient because it's not meant to happen, because dbt is smartest when you let it take responsibility for building everything in the right order, instead of manually ensuring that int_orders doesn't run until after stg_orders is complete (assuming everything is a table)